netlify.toml

```diff
@@ -3,7 +3,7 @@ publish = "public"
 command = "make production-build"
 
 [build.environment]
-HUGO_VERSION = "0.111.3"
+HUGO_VERSION = "0.123.7"
```

Comment:
Unless you _really_ need Hugo 0.123.7, I'd suggest that you upgrade the Hugo version separately to have having too many moving parts. Hugo 0.123.x has a rewritten core, and they're still ironing out the bugs, e.g., see https://github.com/open-telemetry/opentelemetry.io/issues/4064.

Reply:
[I just needed to upgrade to 0.116.0+ to access the "like" operator on the "where" function](https://discourse.gohugo.io/t/where-or-like-operator-not-working-with-netlify-github-workflow/48839/2?u=haddscot)

is there a version you would recommend?